Change to GetIndicatorDBotScore (#12561)
demisto/content: Packs/CommonScripts/Scripts/GetIndicatorDBotScore/README.md

| **Argument Name** | **Description** |
| --- | --- |

> | indicator | he indicator to get the reputation of. Only system indicator types are supported. In order to send multiple indicators you can either use a JSON formatted string representation (e.g `["indicator1", "indicator2"]`). |

Reviewer: ```suggestion
| indicator | the indicator to get the reputation of. Only system indicator types are supported. In order to send multiple indicators, use either a list or a JSON formatted string representation (e.g `["indicator1", "indicator2"]`). |
```
